tsconfig.base.json

```diff
@@ -0,0 +1,32 @@
+// Base TSConfig, used for consistency.
+// Other configurations may extend this file with the "extends" parameter. 
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "resolveJsonModule": true,
+    "esModuleInterop": true,
+    "declaration": true,
+    "target": "es5",
+    "lib": ["es6"],
+    "types": ["node", "mocha", "reflect-metadata"],
+    "moduleResolution": "node",
+    "experimentalDecorators": true,
+    "emitDecoratorMetadata": true,
+    "noImplicitAny": false,
+    "sourceMap": true,
+    "outDir": "dist",
+    "baseUrl": ".",
+    "paths": {
+      "*": [
+        "node_modules/*"
+      ],
+      "@api/*" : [
```

Comment:
We do not use neither `@api` and `@test` aliases anymore. How about we remove them from here?